client/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "openmsupply-client",
-  "version": "1.0.3",
+  "//": "Version is shared between client and remote server and is located in repo root package.json",
+  "version": "0.0.0",
```

Comment:
I could have used client/package.json, but thought it was weird to change client/package.json to increment migration version. 

Btw you'll see further down that this is file is embedded in binary

Reply:
perhaps `0.1.0` to be consistent with server mods? 
The `0.0.0` version doesn't sit well with me.. but I have no good alternative to offer 🤷

Reply:
🙏 53d72db395ca23f05b2f2c2609d6a64bfad7c48c